### PR TITLE
Ember CLI 3.4.4 Update

### PR DESCRIPTION
**Pull request by tomoguisuru**
- Updates to latest LTS, Ember CLI 3.4 (see [Ember Builds](https://www.emberjs.com/builds/))
- Resolves `sendAction` deprecation and updates supporting docs and examples
- Drops support for Ember CLI < 2.18
- Updates test to use most recent testing API

### CODE REVIEW

**Comment by raycohen** [comment]
You might want to call out in the description of this PR that you dropped backwards compatibility for a bunch of versions (granted old versions) of ember that we've supported for a long time.

File: addon/components/labeled-radio-button.js [hunk 1]
```diff
@@ -24,7 +24,11 @@ export default Component.extend({
 
   actions: {
     innerRadioChanged(value) {
-      this.sendAction('changed', value);
+      if (!this.changed || typeof this.changed !== 'function') {
```
Comment (raycohen) [inline addon/components/labeled-radio-button.js]: this commit should be a separate PR, and it should come with a major version bump and updates to the readme

Comment (raycohen) [reply addon/components/labeled-radio-button.js]: that major version bump can be combined with dropping ember support before the current LTS.

Comment (tomoguisuru) [reply addon/components/labeled-radio-button.js]: I've removed the commit but this version is still only compatible with Ember CLI 2.18+

I've updated the readme to include 2 two versions of the install instructions, one for < 2.18 and one for 2.18+

I'll submit another PR with the changes to use closure actions after any issues with this PR have been resolved

**Comment by tomoguisuru** [comment]
@raycohen version has been been bumped to `2.0.0` and supporting docs have been updated to reflect the new requirements

**Comment by raycohen** [comment]
@tomoguisuru From the changes to use closure actions, it looks like this would be possible to introduce in a non-breaking way. When `changed` is a string, use `sendAction`, and when it is a function, call it.
Do you mind opening a new separate PR that only makes that change, and doesn't touch any ember-cli related config, and doesn't bump the package.json version?

**Comment by tomoguisuru** [comment]
@raycohen sure, opened a PR with those changes here, #89

**Comment by tomoguisuru** [comment]
@raycohen the PR has been updated with the changes from master and we should now be good to fully deprecate anything < 2.18 and switch to just closure actions

**Comment by raycohen** [comment]
@tomoguisuru appreciate the work and we will likely want most of these changes, but from a library perspective I don't think we want to drop support for everything all the way up to 2.18. 
For two reasons: First, we can likely support all versions back to 2.4 with the same code anyways. Second is as a utility addon, we're not gaining much by dropping support, and this library is getting enough downloads that I'd much rather err on the side of supporting everything we can.

I am in agreement that we should do a major version bump to 2.0, and it should drop support for at a minimum 1.11, 1.12. and 1.13. I'm on the fence about whether we should support 2.0, 2.4, or 2.8 as the furthest back version we include in our tests.
